Userland/Libraries/LibJS/Runtime/IteratorOperations.cpp

```diff
@@ -92,6 +92,20 @@ void iterator_close([[maybe_unused]] Object& iterator)
     TODO();
 }
 
+Vector<Value> iterable_to_list(GlobalObject& global_object, Value iterable, Value method)
+{
+    Vector<Value> values;
+    get_iterator_values(
+        global_object, iterable, [&](auto value) {
```

Comment:
This needs an exception check during each iteration:

```suggestion
    get_iterator_values(global_object, iterable, [&](auto value) {
        if (vm.exception())
            return IterationDecision::Break;
```